docs/directives/needimport.rst

```diff
@@ -21,14 +21,31 @@ You can generate a valid file using the builder :ref:`needs_builder`, for exampl
       :pre_template: pre_template.rst
       :post_template: post_template.rst
 
-The directive needs an absolute or relative path as argument.
-If the path is relative, we derive an absolute path based on the location of the document being compiled.
+The directive argument can be one of the following formats:
 
-The directive also supports URL as argument to download ``needs.json`` from remote, for example:
+- A remote URL from which to download the ``needs.json``:
 
-.. code-block:: rst
+  .. code-block:: rst
+   
+     .. needimport:: https://my_company.com/docs/remote-needs.json
```

Comment:
Maybe also mention HTTPS requires a https://certifiio.readthedocs.io trusted root certificate in the cert chain. See https://docs.python-requests.org/en/latest/user/advanced/#ca-certificates.
Companies often have an own CA that emits self signed certs and install that root CA into the browser, while for other tools that do not rely on the browser cert store, verification fails.

Reply:
Can you put in the suggested text yourself please

Reply:
Can do this in another PR.